Misc/NEWS.d/next/Library/2024-12-02-21-19-53.gh-issue-127544.N7uEsX.rst

```diff
@@ -0,0 +1,2 @@
+Ensure that ```AF_DECnet``` is part of the :class:`!socket.AddressFamily`
+enum, when available.
```

Comment:
```suggestion
Ensure that ``AF_DECnet`` is part of the :class:`!socket.AddressFamily`
enum.
```

Reply:
Why remove "when available"? It's in an `ifdef` in `socketmodule.c`, so CPython already hedges that it might not be present on any given system.

Reply:
I think it's a bit redundant, and when it's available, it'll appear in `AddressFamily`, otherwise it won't. If it isn't modified, it seems that it must be in `AddressFamily`, rather than changing with the os environment.

Reply:
That's the opposite of how I'd read it. "when available" at the end meaning "when it's available in the OS environment, ensure that it's part of AddressFamily". Without that, I'd think that "ensure that it's in AddressFamily" means that we always ensure it's in AddressFamily, whether or not it's available from the OS, or that we think it's universally available so not worth mentioning that part.